packages/components/src/SidePanel/SidePanel.component.js

```diff
@@ -17,8 +17,10 @@ import theme from './SidePanel.scss';
  ];
  <SidePanel
 	 actions={ actions }
-	 onToggleDock={ action('Toggle dock clicked') }
 	 docked={ isDocked }
+	 selected= { selectedItem }
```

Comment:
please use label which should be uniq (having same label twince in the sidepanel seems really weird to me) and adding key to my action to support this seems also weird to me;

or why not just selected could be the item ? object === object by reference should works great.